common/src/main/java/org/jboss/hal/testsuite/page/BasePage.java

```diff
@@ -224,6 +225,19 @@ public NotificationCenterFragment getNotificationArea() {
         return notification;
     }
 
+    /**
+     * @return returns area display message as popup element in center of page, Area is closed after few seconds automatically usually.
+     */
+    public AlertFragment getAlertArea() {
+        Graphene.waitGui().until().element(By.cssSelector(AlertFragment.ROOT_SELECTOR)).is().visible();
```

Comment:
Can you define selector like `By alertSelector = By.cssSelector(AlertFragment.ROOT_SELECTOR);` and then use it twice instead of code duplication? It would be more readable and maintanable IMO.